examples/tests/doublefree.py

```diff
@@ -43,7 +45,8 @@ def test(self):
         expected_exit_status = -signal.SIGABRT
         output = cmd_result.stdout + cmd_result.stderr
         self.assertEqual(cmd_result.exit_status, expected_exit_status)
-        self.assertIn('double free or corruption', output)
+        self.assertTrue(self.GLIBC_MSG in output or self.MAC_LIBC_MSG in output,
```

Comment:
OK.  An even more assertive way would be checking for only once, depending on the current platform.  But this is fine.